packages/@react-aria/select/src/useSelect.ts

```diff
@@ -68,6 +68,25 @@ export function useSelect<T>(props: AriaSelectOptions<T>, state: SelectState<T>,
     ref
   );
 
+  let onKeyDown = (e: KeyboardEvent) => {
+    switch (e.key) {
+      case 'ArrowLeft': {
+        let key = state.selectedKey ? delegate.getKeyAbove(state.selectedKey) : delegate.getFirstKey();
```

Comment:
We should check if these are null explicitly. We've had issues before where someone might use e.g. an empty string, or zero as a key.

```suggestion
        let key = state.selectedKey != null ? delegate.getKeyAbove(state.selectedKey) : delegate.getFirstKey();
```

Reply:
makes sense done